python/triton/compiler.py

```diff
@@ -905,40 +905,34 @@ def add_external_libs(mod, libs):
     _triton.add_external_libs(mod, list(libs.keys()), list(libs.values()))
 
 
-def ttgir_to_llir(mod, extern_libs):
+def ttgir_to_llir(mod, extern_libs, compute_capability):
     if extern_libs:
         add_external_libs(mod, extern_libs)
     return _triton.translate_triton_gpu_to_llvmir(mod)
```

Comment:
`compute_capability` should be passed to `translate_triton_gpu_to_llvmir`. I can do it in another PR though.

Reply:
Let me add.